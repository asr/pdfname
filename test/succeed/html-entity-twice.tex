\documentclass{article}

\pdfinfo{%
  /Author (\&omega;alpha\&omega;)%
  /Title  (\&omega;alpha\&omega;)%
  /CreationDate (D:2017)%
}

\begin{document}
pdfname test.
\end{document}
