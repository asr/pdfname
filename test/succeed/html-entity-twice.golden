ret > ExitSuccess
out > The full path name will be /tmp/omegaalphaomega-2017.omegaalphaomega.pdf
out >
